Add initial support for PHP7

dgoujard opened the pull request:
To test the environment : 
```
fission env create --name php7 --image damien/fission-php-env
echo '<?php echo "Hello World"; ?>' > myfunction.php
fission function create --name hellophp --code monscript.php --env php7
fission route create --method GET --url /hellophp --function hellophp
curl http://$FISSION_ROUTER/hellophp
```

soamvasani commented:
Thanks for this, @dgoujard! Taking a look.

soamvasani commented on environments/php7/Dockerfile:
```diff
@@ -0,0 +1,29 @@
+FROM php:7.1-cli
+
+RUN apt-get update && apt-cache search libjpeg && apt-get install -y libjpeg-dev && apt-get install -y \
```
what's the apt-cache search for?

dgoujard replied:
apt-cache is for searching package. it's not necessary. I will remove this command.

soamvasani commented on environments/php7/Dockerfile:
```diff
@@ -0,0 +1,29 @@
+FROM php:7.1-cli
```
Would the php7.1-alpine image work?  If all else is equal it would be better to use a smaller image.

dgoujard replied:
I updated the base image to alpine. Before : 195 MB After : 149 MB

soamvasani commented:
@dgoujard, would you mind also adding an example in the examples directly under examples/php?  Preferably something that shows how to get a POST request body, how to log, etc.

dgoujard commented:
@smadhura I updated my PR with examples and switched to alpine base image. I also handle "handler" function with Request, Response and Logger in parameter. Previous scripts without "handler" function still works

soamvasani commented:
Thanks @dgoujard and sorry for the delay in replying.  Taking a look now.

soamvasani commented on environments/php7/src/Server.php:
```diff
@@ -0,0 +1,64 @@
+                    //If the function as an handler class it will be called with request, response and logger
+                    if(function_exists("handler")){
+                        ob_end_clean();
+                        return handler($request, $response,$logger);
```
So in other environments, we wrap up the request, response, logger, and perhaps other stuff into one "context" class.  This helps because if we want to add anything to the context at some point (e.g. a handle to a DB, a Kubernetes connection, etc.), we can do so without breaking compatibility with existing functions.  

Would this pattern be doable here as well?

soamvasani commented on examples/php7/stock.php:
```diff
@@ -0,0 +1,40 @@
+<?php
```
Very nice to see a non-hello world example 👍 :)
